soc/xtensa/intel_adsp/common/include/intel_adsp_ipc.h

```diff
@@ -7,10 +7,7 @@
 #include <intel_adsp_ipc_devtree.h>
 #include <zephyr/kernel.h>
 #include <zephyr/device.h>
-#ifdef CONFIG_PM_DEVICE_RUNTIME
```

Comment:
Dropping this `#ifdef` is another, different change that is not described in the commit message.

Reply:
yep ... it is not necessary either, but I thought would be cumbersome having two commits here, one moving /pm/device.h out of the ifdef and another one removing this ifdef + pm/device_runtime.h

But I can change if it if you want.

Reply:
While a separate change should technically require a separate commit, I didn't mean to ask for that. I'm only wondering about the rationale.